2023-10-17/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "fmt"
+
+func concatenate(a, b string) string {
+	return a + b
+}
+
+func swap(a, b string) (x, y string) {
+	x = b
+	y = a
+	return
+}
+
+func main() {
+	a := "el"
+	b := "Jo"
+
+	a, b = swap(a, b)
```

Comment:
Calling the swap function will then return the result from the return type which is x followed by y which is b followed by a